me.eastsun.hackdesk.yml

```diff
@@ -0,0 +1,24 @@
+app-id: me.eastsun.hackdesk
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: hackdesk
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=wayland # Permission needed to show the window
+  - --socket=x11 # Permission needed to show the window
+  - --socket=fallback-x11
+  - --filesystem=home/.hackdesk:create # Permission needed to save the config
+
+modules:
+  - name: hackdesk
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 hackdesk.AppImage /app/bin/hackdesk
+    sources:
+      - type: file
+        url: https://github.com/EastSun5566/hackdesk/releases/download/hackdesk-v0.0.7/hack-desk_0.0.7_amd64.AppImage
```

Comment:
how does that work on aarch64?

Reply:
Fixed! I add `only-arches: [x86_64]`

Reply:
an aarch64 package will still be created. Why not building aarch64?